src/hnsw.cc

```diff
@@ -38,6 +39,8 @@ using std::endl;
 using std::fstream;
 using std::max;
 using std::min;
+using std::fmax;
+using std::fmin;
```

Comment:
It does not needed since you use fmin/fmax with std namespace explicitly.

Reply:
I was actually a bit curious about this entire block. Easy fix.